woocommerce/woocommerce-gutenberg-products-block
# Remove API summaries in favour of client side code

In 2378 I found that we were not using the Product Summary PHP class on the frontend; only in API responses consumed by the client side code.

Due to this, I think it's better to move this logic to the client side, and use the short/long description instead. This will give us more control over description rendering without additional calls to the API to get an updated summary.

This is an alternative solution to https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/2378 which Fixes #2293.

### How to test the changes in this Pull Request:

1. Test atomic product summary block within All Products. Summary word limit is 150.
2. Check items in cart/checkout. Summary word limit is 15.
